plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -89,6 +89,21 @@ open class HtmlRenderer(
                 childrenCallback()
                 if (node.hasStyle(TextStyle.Monospace)) copyButton()
             }
+            node.dci.kind == ContentKind.Parameters -> {
+                span("parameters $additionalClasses") {
+                    childrenCallback()
+                }
+            }
+            node.dci.kind == ContentKind.Parameter -> {
+                span("parameter $additionalClasses") {
+                    if (node.hasStyle(ContentStyle.Indented)) {
+                        // could've been done with CSS (padding-left), but the indent needs to
```

Comment:
I don't like having extra spaces in copied text. 

Have you tried this!?
```css
.intented::before {
    content: '    '
}
```

Reply:
Me neither, but unfortunately both padding and content don't work with select or copy. I added `content: '    '`, here's the result (it's now 8 spaces, 4 physical and 4 from content)

![indented-ff](https://user-images.githubusercontent.com/22685910/149754240-0c56e65a-3cb7-4829-b57f-9797be5c4f3f.png)

Reply:
Also, I'm not sure what you mean by "I don't like having extra spaces in copied text."

Would you like the constructor to be copied like this?

```
class PrimaryConstructorWithVals(
val valString: String = "val def", 
var varString: String = "var def", 
var valStringWithoutDefaultValue: String, 
paramString: String
)
```

If so, content will do just fine then, but I don't think it's right

Reply:
Discussed in person. In chrome everything works as expected, whereas Firefox does copy text without line breaks, so it results in
```
class PrimaryConstructorWithVals(    val valString: String = "val def",     var varString: String = "var def",     var valStringWithoutDefaultValue: String,     paramString: String)
```

Reply:
It seems to be a firefox-specific bug. Created an issue for this: https://github.com/Kotlin/dokka/issues/2314